flexmeasures/conftest.py

```diff
@@ -156,6 +162,23 @@ def create_test_accounts(db) -> dict[str, Account]:
     )
 
 
+def add_consult_account_access_to_prosumer_account(db, test_accounts):
```

Comment:
It seems this logic could just be inside `create_test_accounts()`, as it is used in both functions that call it - and it creates an account.

Reply:
updated, removed the extra function and added the function to `create_test_accounts()`